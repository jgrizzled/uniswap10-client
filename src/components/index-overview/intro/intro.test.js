import Intro from '.';

const component = <Intro />;

// smoke test
it('renders without crashing', async () => {
  const div = document.createElement('div');
  ReactDOM.render(<Providers>component</Providers>, div);
  ReactDOM.unmountComponentAtNode(div);
});

// shallow snapshot test
it('renders as expected', async () => {
  const wrapper = Enzyme.shallow(component, { wrappingComponent: Providers });
  expect(wrapper).toMatchSnapshot();
});
