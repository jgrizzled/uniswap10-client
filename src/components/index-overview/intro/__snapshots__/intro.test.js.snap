// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<styled.div>
  <div>
    <h2>
      Track the top trading tokens on Uniswap
    </h2>
    Uniswap10 is an index of the top 10 traded tokens on 
    <a
      href="https://uniswap.io/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Uniswap
    </a>
    .
  </div>
  <div>
    <h2>
      Invest in liquid DeFi assets
    </h2>
    Tokens are ranked by market activity. 
    <Link
      to="/faq"
    >
      Learn more
    </Link>
    .
  </div>
  <div>
    <h2>
      Customize it!
    </h2>
    Try out the 
    <Link
      to="/settings"
    >
      settings
    </Link>
    .
  </div>
</styled.div>
`;
